src/components/molecules/AmountPicker.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+import Radio from '@material-ui/core/Radio'
+import RadioGroup from '@material-ui/core/RadioGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import FormControl from '@material-ui/core/FormControl'
+import FormLabel from '@material-ui/core/FormLabel'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+
+type Props = {
+  selectedValue: string
+  handleAmountChange: (e: React.ChangeEvent<any>) => void
+  label: string
+  optionsCount: number
+}
+
+const renderOptions = (optionsCount: number) =>
+  Array.from(Array(optionsCount).keys()).map(i => {
```

Comment:
What's i in this case? The label, right?

Reply:
`i` is simply a number. What I do here is the following:

* create an array with the length of optionscount, it looks like this: `[ undefined, undefined, ... ]`
* because we care for its indices (0, 1, 2, 3 etc.) and not `undefined` we get its keys
* the resulting array is: `[0, 1, 2, 3, ...]`
* now we can map over this array and build our option labels
* because our options should start at _1_ not at _0_ we add _1_